remove-eb-migration-transient.php

```diff
@@ -0,0 +1,207 @@
+     *
+     * @since 1.0.0
+     *
+     * @return bool true if successful, false otherwise
+     */
+    public function maybe_delete_old_transient() {
+
+        if ( ! $this->old_transient_exists() ) {
+            add_option( self::$deleted_key, 'false' );
+            return false;
+        }
+
+        $deleted = delete_transient( self::$transient_key );
+
+        if ( $deleted ) {
+            add_option( self::$deleted_key, 'true' );
+        }
+
+        return $deleted;
+    }
+
+    /**
+     * After the extension's deleted under this exte plugin row.
+     *
+     * @since 1.0.0
+     *
+     * @return bool true if successful, false otherwise
+     */
+    public function maybe_add_deactivation_notice( $plugin_file, $plugin_data, $status ) {
+
+        if ( false === get_option( self::$deleted_key ) ) {
+            return false;
+        }
+
+        $message  = esc_html__( 'No Eventbrite Tickets cleanup task was necessary.', 'tribe-extension' );
+        $dashicon = 'dashicons-flag';
+
+        if ( 'true' === get_option( self::$deleted_key ) ) {
+            $message  = esc_html__( 'Success! Eventbrite\'s migration data was cleaned up.', 'tribe-extension' );
+            $dashicon = 'dashicons-yes';
+        }
+
+        printf(
+            '<tr class="active tribe--plugin-row-notice"><th class="check-column" scope="row"><span class="dashicons %1$s"></span></th><td><strong>%2$s</strong></td><td>%3$s</td></tr>',
+            $dashicon,
+            $message,
+            esc_html__( 'You can now deactivate this extension, then delete it.', 'tribe-extension' )
+        );
+    }
+
+    public function admin_head_css() {
+        ?><style>
+        tr[data-plugin="remove-eb-migration-transient.php"] td,
+        tr[data-plugin="remove-eb-migration-transient.php"] th {
+            box-shadow: none !important;
+        }
+
+        tr.tribe--plugin-row-notice .check-column {
+            text-align: center;
+        }
+
+        tr.tribe--plugin-row-notice .dashicons {
+            margin-left: 5px;
+            line-height: 1.25;
+        }
+
+        /* Convoluted selectors, but adds a mouseover effect that helps the two separate table rows appear as one unit. */
+        tr.active[data-plugin="remove-eb-migration-transient.php"]:hover td,
+        tr.active[data-plugin="remove-eb-migration-transient.php"]:hover th,
+        tr.active[data-plugin="remove-eb-migration-transient.php"]:hover + tr td,
+        tr.active[data-plugin="remove-eb-migration-transient.php"]:hover + tr th {
+            background: #fdfff2 !important;
+        }
+        </style><?php
+    }
+
+    /**
+     * Remove deletion flag from options upon deactivation.
+     *
+     * @since 1.0.0
+     *
+     * @return bool true if successful, false otherwise
+     */
+    public static function cleanup_on_deactivation() {
+        return delete_option( Tribe__Extension__Remove_Eventbrite_Migration_Transient::$deleted_key );
```

Comment:
use `self::$deleted_key` here, no need for the FQN name.

Reply:
Good catch, this was a typo.